bot/internal/bot/label.go

```diff
@@ -36,7 +40,14 @@ func (b *Bot) Label(ctx context.Context) error {
 		return trace.Wrap(err)
 	}
 
-	labels, err := b.labels(ctx, files)
+	v, err := b.c.GitHub.GetLatestReleaseMajorVersion(ctx,
+		b.c.Environment.Organization,
+		b.c.Environment.Repository)
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
Can this be done inside `b.labels()` and only on documentation PRs to avoid extra API call for all other PRs? Then you also won't need to change the `b.labels()` method contract.

Also, when you move it, give variable a meaningful name instead of just `v`.